api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

```diff
@@ -234,7 +253,7 @@ message TcpProxy {
   // <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.TcpAccessLogOptions.access_log_flush_interval>`.
   // Note that if both this field and :ref:`access_log_flush_interval
   // <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.TcpAccessLogOptions.access_log_flush_interval>`
-  // are specified, the former (deprecated field) is ignored.
+  // are specified, an exception will be thrown.
```

Comment:
The former behavior should be kept. The motivation is for deployments where a fleet of both old and new Envoys live, and the management server can send both the old and new fields, so the clients can still function properly.